dropwizard-testing/src/main/java/io/dropwizard/testing/DropwizardTestSupport.java

```diff
@@ -81,7 +92,21 @@ public DropwizardTestSupport(Class<? extends Application<C>> applicationClass, S
      *   be manipulated in any way, no overriding
      */
     public DropwizardTestSupport(Class<? extends Application<C>> applicationClass,
-            C configuration) {
+                                 C configuration) {
+        this(applicationClass, configuration, ServerCommand::new);
+    }
+
+
+    /**
+     * Alternate constructor that allows specifying the command the Dropwizard application is started with.
+     * @since 1.1.0
+     * @param applicationClass Type of Application to create
+     * @param configuration Pre-constructed configuration object caller provides; will not
```

Comment:
Are you thinking a `DropwizardTestSupportConfig` POJO for all this instead of constructor overloading?